CONTRIBUTING.adoc

```diff
@@ -38,52 +39,49 @@ These steps should help you setup a development environment for the Globus CLI.
 You should now have, in your virtualenv, the `globus-cli` installed, pointed at
 your local copy of the repo. `globus` will invoke code directly out of the
 repo.
-You can write and test commands interactively in this setup.
 
-Linting & Testing
------------------
+== Running Tests
+
+We use `tox` to run our tests against all supported Python versions.
+
+You can use a tool like `pyenv` to install all of the Python versions, or
+install `uv` and the `tox-uv` plugin to have `uv` manage the interpreter versions for you.
+(`tox-uv` will also use `uv` to provision test environments.)
+
+For example:
+
+    uv tool install tox --with=tox-uv
```

Comment:
Could I recommend adding `pipx` as well, with the `tox-uv` injection?

```
pipx install tox
pipx inject tox tox-uv
```

Reply:
I'll add this, but I'll note the rationale for `uv tool` as the only documented method:

- `uv tool` seemed to match best to the _typical_ contributor who might want to use `tox-uv`.
- Documenting one example is good. Two is okay. Documenting many (e.g., 5) would obviously be too much. I tried to stick to just one.
- I'm trying to keep my biases out of our common docs. My personal setup is `pipx install tox --suffix '_uv', pipx inject tox_uv tox-uv`, which lets me switch back and forth between `tox` and `tox_uv` freely.